src/server/middleware/repoproxy/proxy.go

```diff
@@ -86,13 +86,13 @@ func preCheck(ctx context.Context) (art lib.ArtifactInfo, p *models.Project, ctl
 // ManifestGetMiddleware middleware handle request for get manifest
 func ManifestGetMiddleware() func(http.Handler) http.Handler {
 	return middleware.New(func(w http.ResponseWriter, r *http.Request, next http.Handler) {
-		if err := handleManifest(w, r, next); err != nil {
+		if err := handleManifest(w, r, next, http.MethodGet); err != nil {
 			httpLib.SendError(w, err)
 		}
 	})
 }
 
-func handleManifest(w http.ResponseWriter, r *http.Request, next http.Handler) error {
+func handleManifest(w http.ResponseWriter, r *http.Request, next http.Handler, method string) error {
```

Comment:
if you only have one func `handleMafniest` you should only need one `ManifestMiddleware` ?

Reply:
Updated